1 Quick Sort/QuickSort1.py

```diff
@@ -0,0 +1,89 @@
+import random
+import time
+import os
+numbersArr = []
```

Comment:
Please remove all the "arr" words inside the file. Leave just ```numbers``` (because we don't use Hungarian notation)